src/or/or.h

```diff
@@ -3261,6 +3282,10 @@ typedef struct origin_circuit_t {
    * associated with this circuit. */
   edge_connection_t *p_streams;
 
+  /** Smartlist of half-closed streams (half_connection_t*) that still
```

Comment:
Typo: half_edge_t* ?

Reply:
Also in df41067eee614182aabcd047f641df9f3e80d8ba